bot_impl.py

```diff
@@ -5725,6 +5749,10 @@ async def on_message(message):
                 else:
                     vt = int(argument == "yes" or argument == "y")
 
+                if voudon_in_play and the_player != voudon_in_play and not the_player.is_ghost and vt > 0:
```

Comment:
I've seen this line at least four times in this review.  Should this be pulled out to a helper function?